config/routes.rb

```diff
@@ -0,0 +1,61 @@
+Rails.application.routes.draw do
+
+
+
+
+  # The priority is based upon order of creation: first created -> highest priority.
+  # See how all your routes lay out with "rake routes".
+
+  # You can have the root of your site routed with "root"
+  root 'cakes#index'
+  resources :cakes
```

Comment:
Please whitelist only actions you actually use - http://guides.rubyonrails.org/routing.html#restricting-the-routes-created